src/solve1.F

```diff
@@ -416,6 +416,7 @@ subroutine solve1(nstep,rbufsz,num_soil_layers,                  &
       else
         tqnudge = .false.
       endif
+      tqnudge = .false.
```

Comment:
@drichte2 This seems problematic to turn tqnudge off by default.  Is this valid for all uses of CM1 or just for CM1 running testcase == 10 and with droplets?